### PR TITLE
add migrations

### PR DESCRIPTION
**Pull request by namframgia**
add migration

### CODE REVIEW

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/2419?issue_count=1&issue_position=1

**Comment by namframgia** [comment]
ready

File: database/migrations/2018_10_04_035141_create_batches_table.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('batches', function (Blueprint $table) {
+            $table->increments('id');
+            $table->date('start_day');
+            $table->date('stop_day');
```
Comment (BunnyPi04) [inline database/migrations/2018_10_04_035141_create_batches_table.php]: stop day để nullable nhé

**Comment by namframgia** [comment]
ready

**Pull request merged**